scripts/release.sh

```diff
@@ -44,32 +44,32 @@ then
   [[ "$0" = "$BASH_SOURCE" ]] && exit 0
 fi
 
-cd ./client || exit -1
-npm version --no-git-tag-version --no-commit-hooks $1 || exit -1
+( cd client
```

Comment:
What's the point of using `(`? :)

Reply:
It helps with operations in different directories, by changing it only in the scope of a subshell. http://www.tldp.org/LDP/abs/html/subshells.html

Reply:
:+1: